django/bossspatialdb/views.py

```diff
@@ -427,7 +507,54 @@ def get_frame(idx):
 
         }
 
-        session = bossutils.aws.get_session()
+        # Check that only administrators are triggering extra large downsamples
+        large_frame = 100000 * 100000 * 3000 # ~ 200 x 200 x 200 cubes
```

Comment:
Should there be something like `settings.DOWNSAMPLE_MAX_SIZE` instead of a hard coded value?